config/config.go

```diff
@@ -69,21 +73,24 @@ type Reloadable interface {
 var cfg *config
 
 type config struct {
-	ID            string `json:"ID"`
-	Email         string `json:"Email"`
-	Environment   string `json:"Environment"`
-	Key           string `json:"Key"`
-	ConfigVersion string `json:"Version"`
+	ID    string
+	Email string
+	Key   string
+
+	ConfigVersion string `json:"version"`
+	Environment   string `json:"environment"`
+	GID           string `json:"groupID"`
+	GroupName     string `json:"groupName"`
 	// TODO: implement an app wide file logger
 	// LogDirectory string `json:"LogDirectory"`
 
-	InsecureSpeedTest bool `json:"InsecureSpeedTest"`
-	NoDiscoverGateway bool `json:"NoDiscoverGateway"`
-	Nonvolatile       bool `json:"Nonvolatile"`
-	PingEnabled       bool `json:"PingEnabled"`
-	QuietSpeedTest    bool `json:"QuietSpeedTest"`
-	RealtimeEnabled   bool `json:"RealtimeEnabled"`
-	SpeedTestEnabled  bool `json:"SpeedTestEnabled"`
+	InsecureSpeedTest bool `json:"insecureSpeedTest"`
```

Comment:
this looks like what was confusing me. i expected capital letters in the json but when i dumped the request body it was lower case or something and i had no idea what to send back

this piece of the test is what applies here: https://github.com/purplesoda/imup/blob/9da9f7d108a2a45e2d4aad9bb3256ebce6bda6d0/realtime/services/mongo.test.js#L31-L41